Resources/Prototypes/Entities/Mobs/Customization/Markings/scars.yml

```diff
@@ -45,7 +45,19 @@
   bodyPart: Chest
   markingCategory: Chest
   speciesRestriction: [Human, Dwarf, Felinid, Oni] #Einstein Engines - Felinid, Oni
+  sexRestriction: [Male] #DeltaV: Splitting the scars and tattoos
   followSkinColor: true
   sprites:
   - sprite: Mobs/Customization/scars.rsi
     state: scar_chest
+
+- type: marking
+  id: ScarChestFemale #DeltaV: Splitting the scars and tattoos
+  bodyPart: Chest
+  markingCategory: Chest
+  speciesRestriction: [Human, Dwarf, Felinid, Oni] #Einstein Engines - Felinid, Oni
+  sexRestriction: [Female] #DeltaV: Splitting the scars and tattoos
+  followSkinColor: true
+  sprites:
+  - sprite: DeltaV/Mobs/Customization/scars.rsi
+    state: scar_chest_female
```

Comment:
At least comment with "#Start DeltaV additions" and "#End DeltaV" or ideally move to deltav namespace